sktm/patchwork.py

```diff
@@ -44,6 +44,12 @@
                     'fail': 3}
 
 
+def join_with_slash(base, suffix):
+    base = base.rstrip('/')
+    suffix = suffix.lstrip('/').rstrip('/')
```

Comment:
You can use `suffix.strip('/')` instead of doing left and right strip, it will be cleaner and lead to same result. That said, I'm not so sure we want to remove the trailing slash.